.github/workflows/pull-request-ipv-stub.yml

```diff
@@ -0,0 +1,110 @@
+name: Pull Request CI - IPV Stub
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" or "develop" branch
+  pull_request:
+    branches:
+      - main
+    types:
+      - opened
+      - reopened
+      - ready_for_review
+      - synchronize
+    paths:
+      - 'f2f-ipv-stub/**'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Required for sam-validate step
+permissions:
+  id-token: write
+  contents: read
+
+env: # Only adding the variables in that are required for 
+  AWS_REGION: eu-west-2
+
+jobs:
+  run-code-check:
+    name: Code Checks
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: f2f-ipv-stub
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Checks if committing secrets to repo
+        uses: pre-commit/action@v3.0.0
+        with:
```

Comment:
so we are checking all files instead of that argument as pre-commit is doing that. extra_args: "detect-secrets --all-files"

Reply:
Sorry, I don't follow, are you asking me to switch this to `detect secrets --all-files`?

Reply:
yes, thats the way we are following now. apology wasn't very clear earlier